install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -3295,10 +3295,10 @@
 
 if (isDatabaseVersion('201603150')) {
 
-	Settings::AddNew("panel.db_version", "201604120");
+	Settings::AddNew("system.ssl_protocols", "TLSv1,TLSv1.1,TLSv1.2");
 	
-	showUpdateStep("Adding ssl-protocols setting");
-	Database::query("INSERT INTO `panel_settings` SET `settinggroup` = 'system', `varname` = 'ssl_protocols', `value` = 'TLSv1,TLSv1.1,TLSv1.2'");
+	showUpdateStep("Adding ssl_protocols setting");
+	Database::query("INSERT INTO `" . TABLE_PANEL_SETTINGS . "` SET `settinggroup` = 'system', `varname` = 'ssl_protocols', `value` = 'TLSv1,TLSv1.1,TLSv1.2'");
```

Comment:
remove this line...and place the Settings::AddNew() here....the insert into is EXACTLY what Settings::AddNew() does